app/models/user.rb

```diff
@@ -1,10 +1,10 @@
 # frozen_string_literal: true
+include UnicodeUtils
```

Comment:
Didn't know that was a thing! I tried ruby's built in and it couldnt handle unicode.
![image](https://cloud.githubusercontent.com/assets/2114323/21697270/8e7304e8-d35f-11e6-8301-c954e4c8dd6d.png)